### DISCUSSION TITLE
trying to install on brand new laptop with Windows 11

### DISCUSSION THREAD

**Discussion by goldrushfever** [artisan-roaster-scope/artisan#1303] [Q&A] [answered]
After unzipping file and trying to run the installer says, "you need to install an app for this task" What app is needed?

**Comment by roasterdave** [comment]
Please provide additional information.  What is the file name you downloaded?   Are you trying to install in a sandbox?  At what point in the install process do you see the reported message?

**Reply by goldrushfever** [reply]
Zip file [artisan-win-2.8.4.zip](https://github.com/artisan-roaster-scope/artisan/releases/download/v2.8.4/artisan-win-2.8.4.zip)

the file pkg downloads fine, unzips fine but as soon as you try to run the installer it immediately has a pop up and says you need "you need to install an app for this task" would you like to search for on at in the store. the store If you click yes it pops up a Microsoft store window and says what you are looking for is not here. lol Gee thanks Microsoft.

It is literally brand-new Laptop out of box and thought I had run all updates but apparently had not so trying that now.

**Comment by goldrushfever** [comment, marked as answer]
Looks to be just that, computer so fresh wouldn't open, all good now and will try configuring to the New Kaliedo M10 tomorrow hopefully. Thanks, and my apologies for jumping the gun lol
